extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -38,7 +38,18 @@ const renderShoppingTagTemplate = (tagData) => (
         }
       ></span>
       <span class="amp-story-shopping-tag-pill-text">
-        {tagData['product-tag-text'] || '$' + tagData['product-price']}
+        {(tagData['product-tag-text'] && (
+          <span class="amp-story-shopping-product-tag-text">
+            {tagData['product-tag-text']}
+          </span>
+        )) ||
+          new Intl.NumberFormat('en-EN', {
+            style: 'currency',
+            currency: 'USD',
+          }).formatToParts(0.0)[0].value +
+            Number(
+              new Intl.NumberFormat('en-EN').format(tagData['product-price'])
+            )}
```

Comment:
Does this work for:
- Currencies that have their symbol after the number ($1,000 vs 1.000€)
- Currencies that have a space between the symbol and the number
?

Also, both the language and currency need to be dynamic.

Reply:
This work will be part of #37246